starter-code/iteration-2/main.ts

```diff
@@ -1,11 +1,42 @@
+interface TasksToDo {
+  tasks: Array<string>;
+  addTask(task: string): number;
+  listAllTasks(): void;
+  deleteTask(task: string): number;
+}
```

Comment:
Instead of puting the interface here write it in another file and import it here